argocd/overlays/moc-infra/applications/envs/moc/smaug/operate-first/meteor-shower.yaml

```diff
@@ -9,8 +9,8 @@ spec:
     namespace: aicoe-meteor
   project: operate-first
   source:
-    path: manifests
-    repoURL: https://github.com/AICoE/meteor
+    path: meteor/overlays/moc/prod
```

Comment:
did you mean `meteor/overlays/moc/smaug` ?

Reply:
fixed, thanks! :+1: